files/medium-editor/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "medium-editor",
+  "repo": "yabwe/medium-editor",
+  "files": {
+    "basePath": "dist/",
+    "include": ["js/medium-editor.min.js", "css/medium-editor.min.css", "css/themes/*.min.css"]
```

Comment:
Please add `./` before each path. This will ensure that the directories will be retained and not stripped.